scripts/syntax.sh

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env bash
+# Usage:
+#   scripts/syntax.sh [-u]
+#
+# Arguments:
+#   -u update snapshot files
 
 set -ex -o pipefail
 
 DUMMY_GRAMMARS=$(find syntaxes/test -name '*-dummy.json' -exec echo "-g {}" \; | tr '\n' ' ')
+ARGS=$(cat<<ARGS
+  -s template.ng 
+  -g syntaxes/template.ng.json $DUMMY_GRAMMARS 
+  -t syntaxes/test/**/*.ts
+ARGS
+)
 
-# Template syntax tests
-yarn vscode-tmgrammar-test \
-  -s template.ng \
-  -g syntaxes/template.ng.json $DUMMY_GRAMMARS \
-  -t "syntaxes/test/**/*.ts"
+# Unit tests
+yarn vscode-tmgrammar-test $ARGS
```

Comment:
I'm a little confused, now that `inline_template.ts` no longer has those inline annotations, how's the unit test different from snapshot test?

Reply:
It's not. I just left the lines in there in case we add unit tests later on. Would you prefer removing it?

Reply:
ah, I see. Let's remove it until it's actually needed. Thank you!

Reply:
👍